README.md

```diff
@@ -19,24 +19,26 @@ How can we get the benefits of working on `master` branch, while still not worry
 
 ## Introducing FlipTheSwitch
 
-With FlipTheSwitch, we can choose different code paths at runtime:
+### Auto-Generated Files:
+
+With 'FTSFlipTheSwitch', we can choose different code paths at runtime:
```

Comment:
This should go back above the title, and just be `With FlipTheSwitch, we can...` again.